src/main/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalControllerV2.java

```diff
@@ -242,4 +246,68 @@ public ResponseEntity<Object> updatesOrganisation(
 
         return updateOrganisationById(organisationCreationRequest, organisationIdentifier);
     }
+
+
+    @Operation(
+        summary = "Updates an Organisation's name",
+        description = "**IDAM Roles to access API** : <br> prd-admin",
+        security = {
+            @SecurityRequirement(name = "ServiceAuthorization"),
+            @SecurityRequirement(name = "Authorization")
+        })
+
+    @ApiResponse(
+        responseCode = "200",
+        description = "Organisation name has been updated",
+        content = @Content(schema = @Schema(implementation = String.class))
+    )
+    @ApiResponse(
+        responseCode = "400",
+        description = "If Organisation request sent with null/invalid values for mandatory fields",
+        content = @Content
+    )
+    @ApiResponse(
+        responseCode = "403",
+        description = "Forbidden Error: Access denied",
+        content = @Content
+    )
+    @ApiResponse(
+        responseCode = "404",
+        description = "No Organisation found with the given ID",
+        content = @Content
+    )
+    @ApiResponse(
+        responseCode = "500",
+        description = "Internal Server Error",
+        content = @Content
+    )
+
+    @PutMapping(
+        value = "/{orgId}/sra",
+        consumes = APPLICATION_JSON_VALUE,
+        produces = APPLICATION_JSON_VALUE
+    )
+    @ResponseStatus(value = HttpStatus.OK)
+    @ResponseBody
+    @Secured({"prd-admin"})
+    public ResponseEntity<Object> updateOrganisationSra(
+        @io.swagger.v3.oas.annotations.parameters.RequestBody(description = "organisationsraUpdateRequest")
+        @Valid @NotNull @RequestBody OrganisationSraUpdateRequest organisationSraUpdateRequest,
```

Comment:
change endpoint to accept list of organisation ids and sra ids similar to update org name endpoint and if any org id is failed to update then return error message for those failed org id's